pkgs/applications/blockchains/crypto-org-wallet.nix

```diff
@@ -20,7 +20,7 @@ in appimageTools.wrapType2 rec {
     ${imagemagick}/bin/convert ${appimageContents}/${pname}.png -resize 512x512 ${pname}_512.png
     install -m 444 -D ${pname}_512.png $out/share/icons/hicolor/512x512/apps/${pname}.png
     substituteInPlace $out/share/applications/${pname}.desktop \
-      --replace 'Exec=AppRun --no-sandbox %U' "Exec=$out/bin/${pname}"
+      --replace 'Exec=AppRun --no-sandbox %U' 'Exec=${pname} %U'
```

Comment:
Out of curiosity, why `%U` is now inside the `Exec`, too?

Reply:
It was there before so I decided to keep it.

It allows you to pass arguments to the executable. See https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s07.html for more info.